orc8r/cloud/helm/orc8r/charts/secrets/values.yaml

```diff
@@ -64,7 +64,8 @@ secret:
         elasticPort: 9200
 
       orchestrator.yml: |-
-        prometheusPushAddresses: "http://orc8r-prometheus-cache:9091/metrics"
+        prometheusPushAddresses:
+          - "http://orc8r-prometheus-cache:9091/metrics"
```

Comment:
you'll have to bump the version of the subchart, the parent chart's dependency version for this subchart, and the parent chart's version